pmd-vm/src/main/java/net/sourceforge/pmd/lang/vm/ast/ASTprocess.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.vm.ast;
+
+import net.sourceforge.pmd.lang.ast.RootNode;
+
+public final class ASTprocess extends AbstractVmNode implements RootNode {
```

Comment:
where does this come from? I don't see the VM grammar being edited…

Reply:
I made it implement RootNode (needed by AST stages).